gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/generator/BundleGeneratorTest.java

```diff
@@ -0,0 +1,41 @@
+package uk.nhs.adaptors.pss.translator.generator;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+import java.io.IOException;
+
+import org.hl7.fhir.dstu3.model.Bundle;
+import org.hl7.fhir.dstu3.model.ResourceType;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import uk.nhs.adaptors.pss.translator.service.FhirIdGeneratorService;
+
+@SpringBootTest
```

Comment:
Please use `@ExtendWith(MockitoExtension.class)` instead of `@SpringBootTest` here. You then just need `@InjectMocks` annotation over `BundleGenerator bundleGenerator` field (you can check how it was done inside PatientTransferRequestValidatorTest for example)